client/src/domains/constants.ts

```diff
@@ -7,6 +7,18 @@ export const MOBILE_MAX_WIDTH = 575.98;
 
 export const PAGE_CNT = 5;
 
+export const MENU_HEIGHT = 50;
+export const TUTORIAL_CONTENTS = {
+  EDITOR: "1. 에디터고 여기서 편집을 할 수 있으며 기본으로 제공되는 양식은 발표의 첫 페이지이며 없애면 이상해져요.",
```

Comment:
This is the editor. You can write freely with Markdown syntax.
You can set the privacy by clicking the lock button.
You can save the content you are creating by clicking Cmd/Ctrl + s or a button.
Press the start slideshow button to start the presentation.
You can add images by dragging them into the editor.